packages/decoupled-kit-health-check/__tests__/GatsbyWordPressHealthCheck.test.ts

```diff
@@ -27,7 +27,7 @@ describe('GatsbyWordPressHealthCheck', () => {
 	it('should pass for a valid PANTHEON_CMS_ENDPOINT and invalid auth', async ({
 		logSpy,
 	}) => {
-		process.env['PANTHEON_CMS_ENDPOINT'] = 'wordpress.test';
+		process.env['PANTHEON_CMS_ENDPOINT'] = 'https://wordpress.test';
```

Comment:
the `PANTHEON_CMS_ENDPOINT` doesn't have the protocol attached in prod which is what is being modeled here. It's important for the health-check to work in that case and add the protocol if needed since this is the default state when spinning up a new BE + FE through the dashboard.